test/server.js

```diff
@@ -17,6 +17,12 @@ const env = require('./environment');
 
 const testOutputPath = `${__dirname}/test_reports/junit/e2e-results.xml`;
 
+const PORT = process.env.PORT || 3000; // 80/443 standard http/https
+
+const KEY_PATH = process.env.KEY_PATH || '/etc/tls-certs/tls.key';
+const CRT_PATH = process.env.CRT_PATH || '/etc/tls-certs/tls.crt';
```

Comment:
Probably I dont understand this fully so asking.. what are the two scenarios where the key & cert are being read from `etc/tls-certs/..` path vs. from the envVar ? is that in case we are mounting them ?

Reply:
`/etc/tls-certs/` is where I am mounting the `secret` values as a `volume` in the [deployment yaml](https://github.com/benjaminapetersen/origin-web-console-smoke-test/pull/29/files#diff-7021071e07893da155e71cbe9bcc543cR35) and the [pod yaml](https://github.com/benjaminapetersen/origin-web-console-smoke-test/pull/29/files#diff-358c0356866e7b139f280e1f9dc33573R22)

If this is non-standard, it probably would be good to update to whatever is correct.